projects/storefrontlib/shared/components/card/card.component.html

```diff
@@ -72,7 +72,7 @@ <h3 *ngIf="content.title" class="cx-card-title">
             *ngSwitchCase="'delete'"
             class="cx-card-link card-link btn-link cx-action-link"
             (click)="delete()"
-            (keydown.enter)="delete()"
+            (keydown.enter)="delete($event)"
```

Comment:
The `delete` method has been modified which is a breaking change. However, `$event` being an optional parameter allows us to pass it only when we are using the specified of Spartacus to work around the breaking change.

Lets use the feature flag system in the DOM (ie. `cxFeatureLevel` directive) to do this.

There's an example of this being used in `configurator-attribute-checkbox.component.html`.

Reply:
You can also take advantage of this to change links to buttons.